### PR TITLE
Avoid division by zero error

### PR DESCRIPTION
**Pull request by federkasten**
This PR fixes division-by-zero error in `signature-count->matrix` when given an empty signature.
I also renamed `signature-count->matrix` to `signature->vector`.

### CODE REVIEW

File: src/clj_deconstruct_sigs/core.clj [hunk 1]
```diff
@@ -147,7 +147,8 @@
      :diff diff
      :error-sum (Math/sqrt (m/esum (m/mul diff diff))) }))
 
-(defn signature-count->matrix [data]
+(defn signature->vector [data]
   (let [counts (map #(get data % 0) trans-patterns)
         sum (apply + counts)]
-    (map (comp double #(/ % sum)) counts)))
+    (if (pos? sum)
```
Comment (iku000888) [inline src/clj_deconstruct_sigs/core.clj]: I think `when` would be better here

**Review by iku000888 — APPROVED** [review]
@federkasten 
Thanks for the great catch!
I left some nitty gritty but the change looks fine.

File: src/clj_deconstruct_sigs/core.clj (hunk 1 above)
Comment (federkasten) [reply src/clj_deconstruct_sigs/core.clj]: Thanks 👍 I have modified.

**Comment by iku000888** [comment]
Thanks!

**Pull request merged**